packages/core/src/components/axes/axis.ts

```diff
@@ -182,6 +141,50 @@ export class Axis extends Component {
 			.attr("aria-hidden", true)
 			.attr("aria-label", `invisible ${axisPosition} ticks`);
 
+		if (!Tools.getProperty(options, "axes", axisPosition, "visible")) {
```

Comment:
```suggestion
        const isAxisVisible = Tools.getProperty(options, "axes", axisPosition, "visible");
		if (!isAxisVisible) {
```

Reply:
Also, as I've requested in my 2 previous change requests, we need to add `.attr("aria-hidden", isAxisVisible);` to the axis element

Reply:
Also, any code that sets the domain of the scale should be running here before the check for visibility, for instance you're skipping this piece
![image](https://user-images.githubusercontent.com/14989804/92941693-b587e600-f41e-11ea-9649-3ea40514f5ab.png)

which means for a zoomable axis you actually won't be able to zoom in if the axis is invisible.

Please make sure that all the axis related functionality still works with an invisible axis.

Reply:
When axis is disabled, we shouldn't let the user to zoom in. The main purpose of zooming in is to have have more detail information about the data. However, if the user has already disabled axis, the chart would no longer serve as a detail provider. So I don't think providing zoom functionality to chart without axis is a good idea. What do you think? @theiliad